### ISSUE TITLE
Can we switch to V8？

### ISSUE THREAD

**Issue by adsadswdas** [RicardooYoung/VersatileTeacher#3]
Hello, I just learned about this field recently. Can I switch to V8? I haven't found any open-source code yet？

**Comment by RicardooYoung** [comment]
Of course! The VT framework is backbone-agnostic, so you can easily adapt it to any backbone.
